pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -43,6 +43,7 @@ class ModelCheckpoint(Callback):
 
         monitor: quantity to monitor.
         verbose: verbosity mode. Default: ``False``.
+        save_last: always saves the model at the end of the epoch. Default: ``False``.
         save_top_k: if `save_top_k == k`,
```

Comment:
what happens if save_top_k is used and the last is also the best, will it write it twice?

Reply:
Sorry, I forgot to answer this last time.

You are correct, the model will be saved twice. I'll address the other concerns in the other conversation =)

Reply:
ok that's ok I guess. I was wondering whether the checkpoints get written to the same file but the filenames are different, I missed that. The test you added clearly shows that.